cmd/routed-eni-cni-plugin/driver/driver.go

```diff
@@ -45,18 +45,25 @@ const (
 	v6DADTimeout = 10 * time.Second
 )
 
+type VirtualInterfaceMetadata struct {
+	IPAddress         *net.IPNet
+	DeviceNumber      int
```

Comment:
why we only have DeviceNumber here but not networkCardNumber

Reply:
We will eventually remove this field once we migrate to v1.20+ 
This is currently added to support downgrade scenarios